src/routes/LoginPage.vue

```diff
@@ -39,6 +39,10 @@
           </v-btn>
         </v-container>
       </v-form>
+      <a
+        href="/register" 
+        style="display: inline-block; width: 100%; text-align: center;"
```

Comment:
Please avoid using `style` tag. Can you share an example of how does it look if the links are aligned left ?

Reply:
it's ugly without and idk how vuetify styles work

Reply:
It is not Vuetify. If you want to use CSS, just use CSS classes.

Reply:
Please look difference on Omega Community discord